test/support/sample/SampleLinkedUnit.scala

```diff
@@ -13,13 +13,23 @@ object SampleLinkedUnit {
       vars = SampleEnterprise.asJson(ern)
     )
 
-  def forVat(period: Period, vatref: VatRef): LinkedUnit =
+  def forVat(period: Period, vatRef: VatRef): LinkedUnit =
     LinkedUnit(
-      UnitId(vatref.value),
+      UnitId(vatRef.value),
       UnitType.ValueAddedTax,
       period,
       parents = Some(Map(UnitType.Enterprise -> UnitId("1234567890"))),
       children = None,
-      vars = SampleVat.asJson(vatref)
+      vars = SampleVat.asJson(vatRef)
+    )
+
+  def forPaye(period: Period, payeRef: PayeRef): LinkedUnit =
+    LinkedUnit(
+      UnitId(payeRef.value),
+      UnitType.PayAsYouEarn,
+      period,
+      parents = Some(Map(UnitType.Enterprise -> UnitId("1234567890"))),
+      children = None,
+      vars = SamplePaye.asJson(payeRef)
     )
 }
```

Comment:
Could we not create a shared method between these and then implement it with forVat and forPaye

Reply:
Following the refactoring to more generic services, these are no longer used.  I will remove them on a subsequent branch.